HacktoberfestProject.Web/Views/Home/Index.cshtml

```diff
@@ -87,3 +87,29 @@ else
     <a asp-action="Add" asp-controller="Home" class="btn btn-outline-primary">Add a PR</a>
   }
 }
+<script type="text/javascript">
```

Comment:
We just need to wrap the `<script></script>` tags in a section :-)

```
@section Scripts {
<script type="text/javascript">
.....
.....
</script>
}
```

Reply:
Done @Layla-P